string-manipulation-1/src/is-upper-cased.ts

```diff
@@ -0,0 +1,14 @@
+/* exported isUpperCased */
+
+function isUpperCased(word: string): boolean {
+  for (let i = 0; i < word.length; i++) {
```

Comment:
Don't need the for loop here. The alternate fn you have inside comments works fine!